fpm/recipes/jenkins-agent/recipe.rb

```diff
@@ -0,0 +1,21 @@
+class Gof3r < FPM::Cookery::Recipe
+  description "The Jenkins swarm plugin enables slaves to auto-discover nearby Jenkins master and join it automatically, thereby forming an ad-hoc cluster."
+  name 'jenkins-agent'
+  homepage 'https://jenkins.io/'
+
+  version '2.2'
+
+  source "https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/swarm-client/#{version}/swarm-client-#{version}-jar-with-dependencies.jar"
+  sha256 'fc5ad10aaca1a3563c1ec650ad9bcb8ecbea0be9cd10053421f89647daeef8eb'
+
+  maintainer 'GOV.UK <govuk-dev@digital.cabinet-office.gov.uk>'
+
+  def build
```

Comment:
I put it there because it's present and empty in other places, including your [own recipe](https://github.com/alphagov/packager/blob/master/fpm/recipes/etcd/recipe.rb#L13-L14).

I'll test building it and see what happens.